apps/web/src/composables/wallet.ts

```diff
@@ -65,9 +70,20 @@ export default function useWallet() {
     }
   }
 
+  // TODO: Fold this into the logic of switching to/from other wallet provider depending on front-end implementation
```

Comment:
Or in the current connectWallet method?

Reply:
Like:
`if (selectedProvider.value && selectedProvider.value === 'WalletConnect') { ... }`